frontend/admin/resources/js/types/react-table-config.d.ts

```diff
@@ -121,7 +121,8 @@ declare module "react-table" {
 
   export interface Cell<
     D extends Record<string, unknown> = Record<string, unknown>,
-    V = any,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    V = unknown,
```

Comment:
Not sure how interfaces are working, and whether this is false positive or something real.

Reply:
fyi only. no needed actions.